data/fields/seamark/type.json

```diff
@@ -4,5 +4,333 @@
+            },
+            "gate": {
+                "title": "Gate",
+                "description": "A structure that may be swung, drawn, or lowered to block an entrance or passageway."
+            },
+            "harbour_basin": {
+                "title": "Harbour Basin",
+                "description": "An enclosed area of water surrounded by quay walls constructed to provide means for the transfer of cargoes from and to ships."
+            },
+            "harbour": {
+                "title": "Harbour",
+                "description": "A Harbour installation with a service or commercial operation of public interest. A body of water providing protection for vessels, and providing anchoring and docking facilities."
+            },
+            "hulk": {
+                "title": "Hulk",
+                "description": "The hull of a wrecked or condemned ship, which is moored in a permanent position or grounded."
+            },
+            "landmark": {
+                "title": "Landmark",
+                "description": "A prominent object at a fixed location which can be used in determining a location or a direction."
+            },
+            "light_major": {
+                "title": "Major Light",
+                "description": "An important luminous aid to navigation, light, lighthouse, to which a name can be attributed. Generally, a light intended for use at sea, usually with a range of 15 miles or more, and in outer approaches to harbours."
+            },
+            "light_minor": {
+                "title": "Minor Light",
+                "description": "A luminous or lighted aid to navigation."
+            },
+            "light_float": {
+                "title": "Light Float",
+                "description": "A boat-like structure used instead of a light buoy in waters where strong streams or currents are experienced, or when a greater elevation than that of a light buoy is necessary."
+            },
+            "light_vessel": {
+                "title": "Light Vessel",
+                "description": "A distinctively marked vessel anchored or moored at a charted point, to serve as an important floating aid to navigation with major light."
+            },
+            "lock_basin": {
+                "title": "Lock Basin",
+                "description": "A lock basin is a wet dock in a waterway, permitting a ship to pass from one level to another."
+            },
+            "marine_farm": {
+                "title": "Marine Farm",
+                "description": "An assemblage of cages, nets, rafts and floats or posts where fish, including shellfish, are artificially cultivated."
+            },
+            "military_area": {
+                "title": "Military Area",
+                "description": "An area within which naval, military or aerial exercises are carried out. Also called an 'Exercise Area'."
```

Comment:
so it is a direct duplicate of `landuse=military`? Why we would wan to promote it or consider it as acceptable to have pointless duplicate tagging scheme?